templates/server/puppetserver/conf.d/ca.conf.erb

```diff
@@ -3,7 +3,7 @@ certificate-authority: {
 
     # settings for the certificate_status HTTP endpoint
     certificate-status: {
-
+<%- if scope.function_versioncmp([@server_puppetserver_version, '2.2']) < 0 -%>
```

Comment:
This appears OK and doesn't trigger the deprecation warning, but the file no longer exists by default in the Puppet Server packages, so ideally it would not be deployed (and `ensure => absent` on new versions, I suppose).